### DISCUSSION TITLE
Record and display time of validation in stats

### DISCUSSION THREAD

**Discussion by diyoyo** [iSoron/uhabits#1471] [Feature Requests]
Hi, I would like to track the consistency of the time I check a certain item. Like, for example : "weigh yourself on the balance". I want to see not only that I did it, but if I'm consistent over multiple days. Is that feasible?
Thanks.

**Comment by hiqua** [comment]
What do you mean by "consistent"? If you click the habit name, you'll see plenty of graphs and stats.
